recipe/meta.yaml

```diff
@@ -30,7 +29,8 @@ test:
   imports:
     - conda_libmamba_solver
   commands:
-    - CONDA_SOLVER_LOGIC=libmamba conda create -n test --dry-run scipy
+    - CONDA_SOLVER_LOGIC=libmamba conda create -n test --dry-run scipy  # [not win]
+    - conda create --experimental-solver libmamba -n test --dry-run scipy  # [win]
 
 about:
```

Comment:
doc_url doesn't resolve. I think it should be https://github.com/conda-incubator/conda-libmamba-solver/blob/main/README.md